hicexplorer/hicFindTADs.py

```diff
@@ -919,8 +920,8 @@ def load_spectrum_matrix(file):
             fields = line.strip().split('\t')
             chrom, start, end = fields[0:3]
             chrom_list.append(chrom)
-            start_list.append(int(start))
-            end_list.append(int(end))
+            start_list.append(int(float(start)))
```

Comment:
the values were like this : 0.0 , 1.0 ... but it was still reading it as string rather than float.. Then threw this error :
```
Error reading line: 1. The start field is not an integer.
Error message: invalid literal for int() with base 10: '0.0'
```